config/custom_items.txt

```diff
@@ -191,6 +191,18 @@ character_name: Belle Day
 item_path: /obj/item/weapon/gun/energy/sizegun
 }
 
+{
+ckey: hottokeeki
+character_name: Belle Day
+item_path: /obj/item/weapon/gun/energy/netgun
+}
+
+{
+ckey: hottokeeki
+character_name: Belle Day
+item_path: /obj/item/weapon/implant/reagent_generator
```

Comment:
`var/datum/reagent/generated_reagent = null`

The default reagent generator doesn't do anything. You need to make a child of it that does.